include/dxc/dxcapi.internal.h

```diff
@@ -160,11 +160,16 @@ struct HLSL_INTRINSIC_ARGUMENT {
               // matching input constraints.
 };
 
+// HLSL_INTRINSIC flags
+static const UINT INTRIN_FLAG_READ_ONLY = 0x1;
+static const UINT INTRIN_FLAG_READ_NONE = 0x2;
+static const UINT INTRIN_FLAG_IS_WAVE = 0x4;
```

Comment:
Personal preference really, but I find defining flags using the bitshift to be clearer once you get to the larger numbers:
```suggestion
static const UINT INTRIN_FLAG_READ_ONLY = 1U << 0;
static const UINT INTRIN_FLAG_READ_NONE = 1U << 1;
static const UINT INTRIN_FLAG_IS_WAVE = 1U << 2;
```